scripts/msct_shape.py

```diff
@@ -23,10 +23,10 @@
 
```

Comment:
maybe don't spend too much time cleaning this function because i've slaughtered it quite a lot already: https://github.com/neuropoly/spinalcordtoolbox/blob/jca/1920-process_seg-csa-perslice/scripts/msct_shape.py